src/shared_modules/utils/windowsHelper.h

```diff
@@ -141,6 +144,80 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(std::string timestamp1, std::string timestamp2)
+    {
+        if (timestamp1.size() == 8)
+        {
+            std::string::iterator it;
+
+            for (it = timestamp1.begin(); it != timestamp1.end(); ++it)
+            {
+                if (!isdigit(*it))
+                {
+                    return UNKNOWN_VALUE;
+                }
+            }
+
+            size_t pos = timestamp2.find(" ");
+
+            if (pos == std::string::npos)
+            {
+                return UNKNOWN_VALUE;
+            }
+
+            std::string date2_trimmed = timestamp2.substr(0, pos);
+            std::string time2_trimmed = timestamp2.substr(pos + 1);
+            date2_trimmed.erase(remove(date2_trimmed.begin(), date2_trimmed.end(), '/'), date2_trimmed.end());
+            time2_trimmed.erase(remove(time2_trimmed.begin(), time2_trimmed.end(), ':'), time2_trimmed.end());
+
+            if (date2_trimmed.size() != 8 || time2_trimmed.size() != 6)
+            {
+                return UNKNOWN_VALUE;
+            }
+
+            tm* local_time_s = new tm();
+            size_t reference_year = 1900;
+
+            if (date2_trimmed.compare(timestamp1) == 0)
+            {
+                return timestamp2;
+            }
+            else
+            {
+                local_time_s->tm_year = stoi(timestamp1.substr(0, 4)) - reference_year;
```

Comment:
It is better to use a `std` namespace method instead of a C function [Link](https://cplusplus.com/reference/string/stoi/)